### DISCUSSION TITLE
[A-L] (2023/24) Foglio 2 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#139] [Esercizi - Bertini]
![Es 2](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/124578df-0429-429e-998d-700f00ef5a2f)

**Comment by daromacarlo** [comment]
![Immagine 2023-10-14 181633](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/707e4154-e3f4-4bc8-9952-7cabae43ce88)

**Comment by niccolo-fato** [comment]
Soluzione analoga a quella di @CarloDaRomadev

**Comment by CiottoloMaggico** [comment]
Analogo a @CarloDaRomadev
